modules/blocks/registration.js

```diff
@@ -10,24 +10,29 @@ const blocks = {};
  *
  * @param {string} slug     Block slug
  * @param {Object} settings Block settings
+ * @return {Boolean}        Whether the block has been successfuly registered
  */
 export function registerBlock( slug, settings ) {
 	if ( typeof slug !== 'string' ) {
-		throw new Error(
+		console.error( // eslint-disable-line no-console
```

Comment:
Would it be less distracting to add a single `/* eslint-disable no-console */` to the top of the file?